AdAway/src/main/res/values-de/strings.xml

```diff
@@ -195,44 +193,44 @@
 
     <!-- Other -->
     <string name="app_subtitle">Werbeblocker</string>
-    <string name="no_text_editor_title">Keine Textbearbeitung installiert</string>
-    <string name="no_text_editor">Es konnte keine Textbearbeitung gefunden werden, um die Hosts-Datei zu öffnen. Sie können Jota oder eine andere Textbearbeitung installieren, um Textdateien zu öffnen.\n
+    <string name="no_text_editor_title">Kein Texteditor installiert</string>
+    <string name="no_text_editor">Es konnte kein Texteditor gefunden werden, um die Hostdatei zu öffnen. Sie können Jota oder andere Texteditoren installieren, um damit zu interagieren. \n
```

Comment:
Should be reverted. Replacing `Textbearbeitung` with `Texteditor` is good, but everything else is worse imho.